src/qibocal/protocols/characterization/two_qubit_interaction/chevron.py

```diff
@@ -0,0 +1,333 @@
+
+
+class ChevronData(DataUnits):
+    """CzFluxTime acquisition outputs."""
+
+    def __init__(self):
+        """Initialize data object with pint."""
+        super().__init__(
+            name="data",
+            quantities={"amplitude": "dimensionless", "duration": "ns"},
+            options=["qubit", "probability", "state", "pair"],
+        )
+
+
+def order_pairs(
+    pair: list[QubitId, QubitId], qubits: dict[QubitId, Qubit]
+) -> list[QubitId, QubitId]:
+    """Order a pair of qubits by drive frequency."""
+    if qubits[pair[0]].drive_frequency > qubits[pair[1]].drive_frequency:
+        return pair[::-1]
+    return pair
+
+
+def create_sequence(
+    ord_pair: list[QubitId, QubitId],
+    platform: Platform,
+    params: ChevronParameters,
+) -> PulseSequence:
+    """Create the experiment PulseSequence for a specific pair.
+
+    Returns:
+        PulseSequence, Dictionary of readout pulses, Dictionary of flux pulses
+    """
+    sequence = PulseSequence()
+
+    ro_pulses = {}
+    qd_pulses = {}
+
+    qd_pulses[ord_pair[0]] = platform.create_RX_pulse(ord_pair[0], start=0)
+    qd_pulses[ord_pair[1]] = platform.create_RX_pulse(ord_pair[1], start=0)
+    fx_pulse = FluxPulse(
+        start=max([qd_pulses[ord_pair[0]].se_finish, qd_pulses[ord_pair[1]].se_finish])
+        + params.dt,
+        duration=params.duration_min,
+        amplitude=1,
+        shape=Rectangular(),
+        channel=platform.qubits[ord_pair[0]].flux.name,
+        qubit=ord_pair[0],
+    )
+
+    ro_pulses[ord_pair[0]] = platform.create_MZ_pulse(
+        ord_pair[0], start=fx_pulse.se_finish + params.dt
+    )
+    ro_pulses[ord_pair[1]] = platform.create_MZ_pulse(
+        ord_pair[1], start=fx_pulse.se_finish + params.dt
+    )
+
+    sequence.add(qd_pulses[ord_pair[0]])
+    sequence.add(qd_pulses[ord_pair[1]])
+    sequence.add(fx_pulse)
+    sequence.add(ro_pulses[ord_pair[0]])
+    sequence.add(ro_pulses[ord_pair[1]])
+    if params.parking:
+        # if parking is true, create a cz pulse from the runcard and
+        # add to the sequence all parking pulses
+        cz_sequence, _ = platform.pairs[
+            tuple(sorted(ord_pair))
+        ].native_gates.CZ.sequence(start=0)
+        for pulse in cz_sequence:
+            if pulse.qubit not in ord_pair:
+                pulse.start = fx_pulse.start
+                pulse.duration = fx_pulse.duration
+                sequence.add(pulse)
+
+    return sequence
+
+
+def save_data(
+    pair: list[QubitId, QubitId],
+    ro_pulses: dict[QubitId, Pulse],
+    results: dict[Union[QubitId, str], AveragedSampleResults],
+    sweep_data: ChevronData,
+    delta_amplitude_range: np.ndarray,
+    delta_duration_range: np.ndarray,
+):
+    """Save results of the experiment in the Data object."""
+    for state, qubit in zip(["low", "high"], pair):
+        ro_pulse = ro_pulses[qubit]
+        result = results[ro_pulse.serial]
+        prob = result.statistical_frequency
```

Comment:
```suggestion
        result = results[measure_target.serial].serialize
        prob = result["0"]
```
Same here I might take a look into the results.py as well